src/Plugin/Field/FieldFormatter/AddToCartFormatter.php

```diff
@@ -34,14 +34,12 @@ public static function defaultSettings() {
    */
   public function settingsForm(array $form, FormStateInterface $form_state) {
     $form = parent::settingsForm($form, $form_state);
-    // @codingStandardsIgnoreStart
-    // $form['combine'] = [
-    //      '#type' => 'checkbox',
-    //      '#title' => t('Combine order items containing the same product variation.'),
-    //      '#description' => t('The order item type, referenced product variation, and data from fields exposed on the Add to Cart form must all match to combine.'),
-    //      '#default_value' => $this->getSetting('combine'),
-    //    ];
-    // @codingStandardsIgnoreEnd
+    $form['combine'] = [
```

Comment:
I think all occurences of "product variation" needs to be replaced with "product bundle"

Reply:
(Or removed) ;)